iceberg/iceberg-handler/src/test/queries/positive/iceberg_major_compaction_partition_evolution2.q

```diff
@@ -29,23 +29,23 @@ use ice_comp;
 create table ice_orc (
     first_name string, 
     last_name string,
-    dept_id bigint
+    is_remote boolean
```

Comment:
why change the test?

Reply:
There are many iceberg compaction q-tests that have partition fields with 'bigint' types, changed this test to test with boolean type. This way iceberg compaction q-tests test more cases, isn't it a good thing?

Reply:
closing as duplicate issue